go.mod

```diff
@@ -28,10 +23,9 @@ require (
 	github.com/cznic/b v0.0.0-20181122101859-a26611c4d92d // indirect
 	github.com/cznic/mathutil v0.0.0-20181122101859-297441e03548 // indirect
 	github.com/cznic/strutil v0.0.0-20181122101858-275e90344537 // indirect
-	github.com/denisenkom/go-mssqldb v0.0.0-20181014144952-4e0d7dc8888f
-	github.com/dgrijalva/jwt-go v0.0.0-20161101193935-9ed569b5d1ac
+	github.com/denisenkom/go-mssqldb v0.0.0-20190121005146-b04fd42d9952
```

Comment:
some of these changes seem unrelated. Could you include only the minimal required changes?